### ISSUE TITLE
entrypoint.sh no proper exception handling.

### ISSUE THREAD

**Issue by f1-outsourcing** [odoo/docker#332]
root@test2:/# ./entrypoint.sh
root@test2:/# echo $?
1

**Comment by lathama** [comment]
Yup there are gaps to work on. I came to work on some problems like this and decided to help clean up the past issues and PRs a bit first.

**Comment by f1-outsourcing** [comment]
good! :)
